tests/integration/multi_table/test_hma.py

```diff
@@ -383,3 +384,138 @@ def test_hma_primary_key_and_foreign_key_only():
     # Assert
     assert all(sample['games']['user_id'].isin(sample['users']['user_id']))
     assert all(sample['games']['session_id'].isin(sample['sessions']['session_id']))
+
+
+def test_use_own_data_using_hma(tmp_path):
+    """End to end test for the ``SDV: Prepare your own data.ipynb``."""
```

Comment:
minor: can we summarize what's being run in this test instead of citing the demo in case the demo changes? Ie.
- loading data
- detecting metadata
- fixing sdtypes 
- etc